packages/neuron-wallet/src/services/addresses.ts

```diff
@@ -140,6 +137,20 @@ export default class AddressService {
     return allGeneratedAddresses
   }
 
+  public static async getNeedGenerateAddressCount(
+    walletId: string,
+    receivingAddressCount: number = DefaultAddressNumber.Receiving,
+    changeAddressCount: number = DefaultAddressNumber.Change
+  ) {
```

Comment:
https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#address-gap-limit

I'd suggest renaming it to `getAddressCountsToFillGapLimit` for better clarity